core/shared/src/main/scala/magnolia.scala

```diff
@@ -446,7 +446,7 @@ object Magnolia {
           }""")
       } else if (isSealedTrait) {
         checkMethod("dispatch", "sealed traits", "SealedTrait[Typeclass, _]")
-        val genericSubtypes = knownSubclasses(classType.get)
+        val genericSubtypes = knownSubclasses(classType.get).toList.sortBy(_.fullName)
```

Comment:
Hmm, does `fullName` include the package? In most cases this would be the same for all subclasses, i.e. redundant.

Reply:
Yes, it includes package name. I realise that in most cases owner name is not relevant but this is a library that is to be used by all sorts of different projects and is entirely possible for projects to nest classes, e.g.
```
sealed trait Animal
object Domestic {
    case object Cat extends Animal
}
object Wild {
    case object Cat extends Animal
}
```
As mentioned in the corresponding issue I think there is some benefit in having ordering consistent with the way shapeless generic does it, but if you would prefer to sort by Symbol name alone that's fine by me.

Reply:
Ah I didn't realize shapeless used `fullName`. Nvm then I think it's good.